src/main/java/spim/fiji/plugin/Image_Fusion.java

```diff
@@ -35,13 +35,13 @@
 
 public class Image_Fusion implements PlugIn
 {
-	public static ArrayList< Fusion > staticFusionAlgorithms = new ArrayList< Fusion >();
+	public final static ArrayList< Fusion > staticFusionAlgorithms = new ArrayList< Fusion >();
```

Comment:
This is a backwards-incompatible change that would require the major version number to increment, as per @StephanPreibisch's pledge (unfortunately hidden to the public in a private email conversation) to heed [SemVer](http://semver.org): `Image_Fusion` is a public class, therefore part of SPIM_Registration's public API, and before this change it is possible to compile this code (while compilation will fail after this change):

```java
// Find that bugger that removed my algorithm
Image_Fusion.staticFusionAlgorithms = new ArrayList< Fusion >( Image_Fusion.staticFusionAlgorithms ) {
    @Override
    public Fusion remove( int i ) {
        throw new RuntimeException( "I do not want anything to be removed!" );
    }
};
```

Note that this is not a theoretical problem, as such code constructs are often needed when highly skilled software developers debug hard problems.